batch/batch/front_end/templates/job.html

```diff
@@ -15,6 +15,28 @@ <h2>Properties</h2>
   <li>Cost: {% if 'cost' in job and job['cost'] is not none %}{{ job['cost'] }}{% endif %}</li>
 </ul>
 
+<h2>Cost Breakdown</h2>
+{% if job['cost_breakdown'] %}
```

Comment:
What causes this to be None? It seems like the DB queries always return it.

Reply:
It's None if the job hasn't started running yet. Same with batches with no jobs running yet.